packages/syft/src/syft/service/worker/worker_service.py

```diff
@@ -317,6 +317,7 @@ def _stop_worker_container(
         container.stop()
         # Remove the container and its volumes
         _remove_worker_container(container, force=force, v=True)
+        return None
```

Comment:
hmm is this needed at all? non-returning should already means `return None`. Did mypy give an error/warning?

Reply:
We can keep this though. Explicitly returning `None` is also good. I'm just curious about if mypy raises a warning for this.

Reply:
Yes @kien without `return None` mypy will give `error: Missing return statement  [return]`